<div align="center">
<img src="Photos/YiyongZhao.jpg" width = "198" height = "288"> 
</div> 

### Curriculum Vitae
<p align = "justify">Yiyong(Rambo) Zhao, currently work as a Postdoctoral Researcher in Bioinformatics at Harvard University. His primary research interests in unveiling the undiscovered functions of the human genome, particularly in the context of neurological diseases. He accomplishes this by applying and refining computational methodologies that incorporate multi-dimensional data from various sources.</p>

<p align = "justify">Rambo earned his Ph.D. in Bioinformatics from Fudan University through a joint Ph.D. program supported by the Chinese Scholarship Council at Pennsylvania State University, USA, in the period from 2019 to 2020. Following this, he gained industry experience as a Bioinformatics & AI Research Scientist at Drug Farm, an innovative drug discovery company in Shanghai led by Dr. Tian Xu, a Chair Genetics Professor at Yale and Vice President at Westlake University. From 2021 to 2022, Dr. Zhao primarily engaged in the discovery of indications and biomarkers and participated in the development of a graph neural network-based method for novel drug target identification. This period in the industry led him to understand his deep-rooted passion for academic research, guiding his transition back to academia. In the academic year 2022-2023, Dr. Zhao held a Professorship at Guizhou University, specializing in crop genetic breeding. He was also recognized as a Class A Professor at the College of Agriculture in Guizhou University during the same year.</p>


<p align = "justify">Rambo has been engaged in gene mining and bioinformatic studies, with over ten papers published in reputed scientific journals such as Nature and Molecular Plant. The publications were chosen as significant basic research results in Shanghai in 2021 and acknowledged as part of the Chinese Science and Technology Journal Excellence Program, forming an integral groundwork for the conservation of crop germplasm resources and breeding. Furthermore, they also established a solid foundation for genomics in molecular cultivation breeding.Dr. Zhao's contributions extend to academic peer review, having evaluated numerous papers for distinguished journals including Molecular Plant, Horticulture Research, Frontiers in Plant Science, Frontiers Molecular Biosciences, Molecular Phylogenetics and Evolution, and Journal of Systematics and Evolution. From 2022 to the present, he serves as an Editorial Board member of Tropical Plants (ISSN 2833-9851), Frontiers in Plant Science, Frontiers in Ecology and Evolution and Innovation Discovery (ISSN 2959-5150).</p>



    Email：yiyong.zhao@yale.edu
    Mobile：+18573815442 
    WechatID：zhaorambo
    Address: 100 College ST, New Haven, CT 06150, USA.

### Education

2016.06 ~ 2021.06：Ph.D. in Bioinformatics, School of Life Sciences, Fudan University, China.

2019.09 ~ 2020.10：Joint Ph.D. Program, Department of Biology, Pennsylvania State University, USA

2012.06 ~ 2016.06：Bachelor of Science in Agriculture (*summa cum laude*), Northeast Forestry University, China

2013.06 ~ 2014.01：Joint Bachelor Program, Kangwon National University, South Korea.



### Work Experience
2024.07 ~ now: Postdoctoral Research Association in Computational Biology, School of Medicine, Section of Biomedical Informatics and Data Science, Yale University, USA.

2023.05 ~ 2024.07: Postdoctoral Research Fellow in Computational Biology, Harvard University, USA.

2022.10 ~ 2023.05: Professor at Department of crop genetic breeding (Special-term Professor Class A), Guizhou University, China

2021.07 ~ 2022.08: Bioinformatics & AI Scientist, at [Drug Farm Co. Ltd](https://drug-farm.com), Shanghai, China.


### Research Interests
<p align = "justify">
I have borad interests in Biological science, including Computational biology, Biological big data mining; Genomics, Revealing the unknown functions of the human genome and their roles in neurological diseases by applying and developing computational methods to integrate multi-dimensional multi-source data, Phylogenomic and comparative genomics; Bioinformatics software development; Genome assembly, Genome evolution such as whole genome duplications; Hybridization detection; Gene evolution and functional genomics.
</p>


### Selected Publications (# Co-first author, * Corresponding author, citations=821, h-index=10)


# [1] __Yiyong Zhao__, Rong Zhang, Kai-Wen Jiang, Ji Qi, Yi Hu, Jing Guo, Renbin Zhu, Taikui Zhang, Ashley N. Egan, Ting-Shuang Yi, Chien-Hsun Huang, and Hong Ma, Nuclear phylotranscriptomics and phylogenomics support numerous polyploidization events and hypotheses for the evolution of rhizobial nitrogen-fixing symbiosis in Fabaceae. Molecular Plant, 2021. 14(5): p. 748-773. (Cover story & Featured article, IF2021=21.949, reported in dozens of domestic and international mainstream media such as Guangming Daily, People's Daily, China News, Science Daily, Phy.org, etc., My contribution: for the cover article, based on the transcriptome and genome, I constructed the largest phylogenetic relationship of Leguminosae to date. Phylogenetic relationships were resolved for five subfamilies, about 30 genome-wide replication events were identified, the evolutionary history of the nitrogen-fixing gene family was explored, and the important role of a new gene in nitrogen fixation was proposed, providing an important basis for germplasm conservation and breeding in the legume family. (The full analyses presented in this article were conducted by me, drafted the original manuscript and made rebuttal.)

#[2] Liangsheng Zhang#, *, Fei Chen#, Xingtan Zhang#, Zhen Li#, __Yiyong Zhao\#__, Rolf Lohaus#, Xiaojun Chang#, Wei Dong, Simon Y. W. Ho, Xing Liu, Aixia Song, Junhao Chen, Wenlei Guo, Zhengjia Wang, Yingyu Zhuang, Haifeng Wang, Xuequn Chen, Juan Hu, Yanhui Liu, Yuan Qin, Kai Wang, Shanshan Dong, Yang Liu, Shouzhou Zhang, Xianxian Yu, Qian Wu, Liangsheng Wang, Xueqing Yan, Yuannian Jiao, Hongzhi Kong, Xiaofan Zhou, Cuiwei Yu, Yuchu Chen, Fan Li, Jihua Wang, Wei Chen, Xinlu Chen, Qidong Jia, Chi Zhang, Yifan Jiang, Wanbo Zhang, Guanhua Liu, Jianyu Fu, Feng Chen, Hong Ma, Yves Van de Peer, and Haibao Tang, The water lily genome and the early evolution of flowering plants. Nature, 2020. 577(7788): p. 79-84. (IF2020=49.962, ESI highly cited paper), (my contribution involved in the assembly, annotation, and correction of transcriptome data of the first aquatic early angiosperm genome. Based on 115 transcriptome and genome-wide data, a highly supported phylogenetic tree of flowering plants was constructed, and molecular clock analysis revealed that early angiosperms originated in the early Cretaceous, while highly supporting the phylogenetic relationships within the Water Lily family. Low-copy nuclear genes from the whole genome confirmed that saprophytic camphor was the earliest differentiated angiosperm. Meanwhile, using a comparative genomics approach, the Blue Star water lily was identified as a potential hybrid parent of two cultivars, laying the foundation for genomics in molecular cultivation breeding. (The related analyses of Phylogenomic relationships and divergence time estimations of angiosperms in Figure 1 were conducted by me and drafted the corresponding part manuscript and made rebuttal.)

#[3] Yezi Xiang#, Taikui Zhang#, __Yiyong Zhao\#__, Hongjin Dong, Hongyi Chen, Yi Hu, Ji Qi, Chien-Hsun Huang, Jun Xiang*, and Hong Ma*. Angiosperm-wide analysis of fruit and ovary evolution aided by a new nuclear phylogeny supports association of the same ovary type with both dry and fleshy fruits. Journal of Integrative Plant Biology, 2024, 00: 1-24. (IF2022=11.4) (My contributions including examined available datasets and chose the high-quality datasets for 423 angiosperm species, representing all major angiosperm lineages and nearly all orders. Due to large scale of hundreds of angiosperm species and many thousands of genes, the molecular evolutionary analyses took a lot of steps and a lot of time. I was able to generate the appropriate gene phylogenies and summarized the results for the purpose of reporting gene duplications at key nodes in the angiosperm phylogeny as reported in this paper.)

*[4] Dandan Li#, Mingjin Huang#, *, Qunwei Han, Dachang Wanag, Kaijie Li, Qiuyue Yang, Ronghui Gu, Guangchong Zhou, Songtao He, Houlin Yu, Lin Cheng, __Yiyong Zhao\*__. A high-quality chromosomal-level reference genome of Dendrobium nobile L. provides new insights into the biosynthesis and accumulation of picrotoxane-type sesquiterpenoid alkaloids, 2024. https://doi.org/10.1016/j.indcrop.2024.118243. Industrial Crops and Products (IF2022=5.9) (My contribution in Conceptualization, Data curation, Investigation, Writing/review/editing & handled revisions and rebuttals.)

*[5] Lin Cheng, Qunwei Han, Fei Chen, Mengge Li, Tiago Santana Balbuena, and __Yiyong Zhao\*__, Phylogenomics as an effective approach to untangle cross-species hybridization event: A case study in the family Nymphaeaceae. Frontiers in Genetics (section in computational genomics), 2022. 13. (IF2022=3.7) (My contributions in conducting nearly all analyses, drafted the original manuscript, and handled revisions and rebuttals.)

*[6] Lin Cheng#, Mengge Li#, Yachao Wang#, Qunwei Han, Yanlin Hao, Zhen Qiao, Wei Zhang, Lin Qiu, Andong Gong, Zhihan Zhang, Tao Li, Shanshan Luo, Linshuang Tang, Daliang Liu, Hao Yin, Song Lu, Tiago Santana Balbuena, and __Yiyong Zhao\*__, Transcriptome-based variations effectively untangling the intraspecific relationships and selection signals in Xinyang Maojian tea population. Frontiers in Plant Science, 2023. 14. (IF2022=5.6) (My contributions in conceptualization and supervision and handled revisions and rebuttals.)

*[7] Daliang Liu, Zhihan Zhang, Yanlin Hao, Mengge Li, Houlin Yu, Xingruo Zhang, Haoyang Mi, Lin Cheng*, __Yiyong Zhao\*__. Decoding the complete organelle genomic architecture of Stewartia sinensis: An early-diverging species in Theaceae with phylogenetic implications. 25, 114 (2024).  BMC Genomics (IF2022=4.4) (My contribution in conceptualization and supervision and handled revisions and rebuttals.)

*[8] Lin Cheng#, Mengge Li#, Qunwei Han, Zhen Qiao, Yanlin Hao, Tiago Santana Balbuena, and __Yiyong Zhao\*__, Phylogenomics resolves the phylogeny of Theaceae by using low-copy and multi-copy nuclear gene makers and uncovers a fast radiation event contributing to tea plants diversity. Biology, 2022. 11(7): p. 1007. (IF2022=4.2) (My contribution in conceptualization and supervision and handled revisions and rebuttals.)

#[9] Yifan Jiang#, *, Fei Chen#, Aixia Song#, __Yiyong Zhao\#__, Xinlu Chen, Yuan Gao, Guo Wei, Wanbo Zhang, Yaqin Guan, Jianyu Fu, Junhao Chen, Wenlei Guo, Zhengjia Wang, Tobias G. Köllner, Zhineng Li, Ning Liu, Shunzhao Sui, Liangsheng Zhang*, and Feng Chen*, The genome assembly of Chimonanthus praecox var. concolor and comparative genomic analysis highlight the genetic basis underlying conserved and variable floral traits of wintersweet. Industrial Crops and Products, 2023. 206: p. 117603. (IF2022=5.9) (The related analyses of phylogenomic relationships and divergence time estimations of magnoliids in Figure 4 were conducted by me and drafted the corresponding part manuscript and made rebuttal.)

[10] Lin Zhang, Xinxin Zhu, __Yiyong Zhao__, Jing Guo, Taikui Zhang, Weichen Huang, Jie Huang, Yi Hu, Chien-Hsun Huang*, and Hong Ma*, Phylotranscriptomics resolves the phylogeny of Pooideae and uncovers factors for their adaptive evolution. Molecular Biology and Evolution, 2022. 39(2). (IF= 16.24) (My contributions including taxon samplings and sequencing, assistance in Lin's data analyses)

[11] Jing Guo, Weibin Xu, Yi Hu, Jie Huang, __Yiyong Zhao__, Lin Zhang, Chien-Hsun Huang, and Hong Ma, Phylotranscriptomics in Cucurbitaceae reveal multiple whole-genome duplications and key morphological and molecular innovations. Molecular Plant, 2020. 13: p. 1-17. (IF=13.164) (My contributions including taxon samplings and sequencing, assistance in Jing's data analyses)

[12] Chengpeng He#, Zhiyu Chen#, __Yiyong Zhao__, Yue Yu, Hongkuan Wang, Cong Wang, Gregory P. Copenhaver, Ji Qi*, and Yingxiang Wang*, Histone demethylase IBM1-mediated meiocyte gene expression ensures meiotic chromosome synapsis and recombination. PLOS Genetics, 2022. 18(2): p. e1010041. (IF2022=5.917) (My contributions including all the gene expression quantification and differential gene expressions analyses from RNA-seq datasets)

[13] Jie Huang, Weibin Xu, Junwen Zhai, Yi Hu, Jing Guo, Caifei Zhang, __Yiyong Zhao__, Lin Zhang, Christopher Martine, Hong Ma, and Chien-Hsun Huang, Nuclear phylogeny and insights into whole-genome duplications and reproductive development of Solanaceae plants. Plant Communications, 2023. 4(4): p. 100595. (IF2022=10.5) (My contributions including taxon samplings and sequencing, assistance in Jie's data analyses)

[14] Yinghai Liang, Shanshan Wang, Chenhui Zhao, Xinwei Ma, __Yiyong Zhao__, Jing Shao, Yuebo Li, Honglian Li, Hongwei Song, Hong Ma, Hao Li, Bingbing Zhang, and Liangsheng Zhang, Transcriptional regulation of bark freezing tolerance in apple (Malus domestica Borkh.). Horticulture Research, 2020. 7(1): p. 205. (IF2020=6.072) (My contributions including the gene expression quantification and differential gene expressions analyses for RNA-seq data)

[15] Zhuang Meng, Jinlei Han, Yujing Lin, __Yiyong Zhao__, Qingfang Lin, Xiaokai Ma, Jianping Wang, Muqing Zhang, Liangsheng Zhang, Qinghui Yang, and Kai Wang, Characterization of a Saccharum spontaneum with a basic chromosome number of x = 10 provides new insights on genome evolution in genus Saccharum. Theoretical and Applied Genetics, 2020. 133(1): p. 187-199. (IF2020=5.699) (My contributions including Phylogeny and divergence time estimation of Poaceae as in Figure 5.)

[16] Weichen Huang, Lin Zhang, J. Travis Columbus, Yi Hu, __Yiyong Zhao__, Lin Tang, Zhenhua Guo, Wenli Chen, Michael McKain, Madelaine Bartlett, Chien-Hsun Huang, De-Zhu Li, Song Ge, and Hong Ma, A well-supported nuclear phylogeny of Poaceae and implications for the evolution of C4 photosynthesis. Molecular Plant, 2022. 15(4): p. 755-777. (IF2022= 27.5) (My contributions including taxon samplings and sequencing, assistance in Weichen's data analyses)

[17] Fanjuan Meng, Li Liu, Mu Peng, ZhongKui Wang, Chao Wang, and __Yiyong Zhao__, Genetic diversity and population structure analysis in wild strawberry (Fragaria nubicola L.) from Motuo in Tibet Plateau based on simple sequence repeats (SSRs). Biochemical Systematics and Ecology, 2015. 63: p. 113-118. (IF2015=1.381) (My contributions including taxon samplings and sequencing, assistance in Mu's data analyses)

[18] Ye Zheng, __Yiyong Zhao__, Lihua Wu, Juan Lin. Study on copy number variation and differentiated expression pattern of CaCA family genes in Capsella. 2020. http://www.jns.fudan.edu.cn/#/digest?ArticleID=1178, Chinese Journal of Fudan University (Natural Science) [J], 59, No.1:1-31 (My contributions including the gene expression quantification and differential gene expressions analyses for RNA-seq data)

[20] Lin Cheng#, Yanlin Hao#, Qunwei Han, Zhen Qiao, Mengge Li, Daliang Liu, Hao Yin, Tao Li, Wen Long, Shanshan Luo, Ya Gao, Zhihan Zhang, Houlin Yu, Xinhao Sun, __Yiyong Zhao\*__. The origin and evolution of orphan genes: a case study in tea plant family. bioRxiv 2024.02.01.578514; doi: https://doi.org/10.1101/2024.02.01.578514 (Communications Biology, IF2022=5.9, under revision) My contribution in conceptualization and supervision and handled revisions and rebuttals.)





1. __ZHAO Y__, ZHANG R, JIANG K, QI J, HU Y, GUO J, ZHU R, ZHANG T, EGANA N, YI T-S, HUANG C-H, MA H 2021. Nuclear phylotranscriptomics/phylogenomics support numerous polyploidization events and hypotheses for the evolution of rhizobial nitrogen-fixing symbiosis in Fabaceae. **Molecular Plant** [J]. 2021, 14(5): 748-773 (First author, **IF= 21.949**, Cover story & Featured article, the study reported in dozens of domestic and international mainstream media such as Guangming Daily, People's Daily, China News, Science Daily, Phy.org, etc., Contribution: for the cover article, based on the transcriptome and genome, we constructed the largest phylogenetic relationship of Leguminosae to date. Phylogenetic relationships were resolved for five subfamilies, about 30 genome-wide replication events were identified, the evolutionary history of the nitrogen-fixing gene family was explored, and the important role of a new gene in nitrogen fixation was proposed, providing an important basis for germplasm conservation and breeding in the legume family.)

2. ZHANG L#\*, CHEN F#, ZHANG X#, LI Z#, **ZHAO Y#**, LOHAUS R#, CHANG X#, DONG W, HO S Y W, LIU X, SONG A, CHEN J, GUO W, WANG Z, ZHUANG Y, WANG H, CHEN X, HU J, LIU Y, QIN Y, WANG K, DONG S, LIU Y, ZHANG S, YU X, WU Q, WANG L, YAN X, JIAO Y, KONG H, ZHOU X, YU C, CHEN Y, LI F, WANG J, CHEN W, CHEN X, JIA Q, ZHANG C, JIANG Y, ZHANG W, LIU G, FU J, CHEN F, MA H, VAN DE PEER Y, TANG H 2020. The water lily genome and the early evolution of flowering plants. **Nature** [J], 577: 79-84. (Co-first author, **IF= 49.962**, ESI highly cited paper, Contribution: Involved in the assembly, annotation and correction of transcriptome data of the first aquatic early angiosperm genome. Based on 115 transcriptome and genome-wide data, a highly supported phylogenetic tree of flowering plants was constructed, and molecular clock analysis revealed that early angiosperms originated in the early Cretaceous, while highly supporting the phylogenetic relationships within the Water Lily family. Low-copy nuclear genes from the whole genome confirmed that saprophytic camphor was the earliest differentiated angiosperm. Meanwhile, using a comparative genomics approach, the Blue Star water lily was identified as a potential hybrid parent of two cultivars, laying the foundation for genomics in molecular cultivation breeding.)

Angiosperm-wide analysis of fruit and ovary evolution aided by a new nuclear phylogeny supports association of the same ovary type with both dry and fleshy fruits. Yezi Xiang#, Taikui Zhang#, Yiyong Zhao#, Hongjin Dong, Hongyi Chen, Yi Hu, Ji Qi, Chien-Hsun Huang, Jun Xiang*, and Hong Ma*. Journal of Integrative Plant Biology, 2024, 00: 1-24. (IF2022=11.4) (My contributions including examined available datasets and chose the high-quality datasets for 423 angiosperm species, representing all major angiosperm lineages and nearly all orders. Due to large scale of hundreds of angiosperm species and many thousands of genes, the molecular evolutionary analyses took a lot of steps and a lot of time. I was able to generate the appropriate gene phylogenies and summarized the results for the purpose of reporting gene duplications at key nodes in the angiosperm phylogeny as reported in this paper.)

3. Lin Cheng#, Mengge Li#, Yachao Wang#, Qunwei Han, Yanlin Hao, Zhen Qiao, Wei Zhang, Lin Qiu, Andong Gong, Zhihan Zhang, Tao Li, Shanshan Luo, Linshuang Tang, Daliang Liu, Hao Yin, Song Lu, Tiago S.Balbuena and __Yiyong Zhao\*__. Transcriptome-based Variations Effectively Untangling the Intraspecific Relationships and Selection Signals in Xinyang Maojian Tea Population. 2023 Volume 14 doi: 10.3389/fpls.2023.1114284. **Frontiers in Plant Science**(section in Plant Bioinformatics). (**IF=6.627**)

4. CHENG L, CHEN F, HAN Q, LI M, Tiago Santana Balbuena, __ZHAO Y\*__. Phylogenomics as an effective approach to untangle cross-species hybridization event: a case study in the family Nymphaeaceae. 2022. 13. **Frontiers in Genetics**(section in computational genomics). (**IF=4.772**)

5. CHENG L#, LI M#, HAN Q, QIAO Z, HAO Y, Tiago Santana Balbuena, __ZHAO Y\*__. Phylogenomics Resolves the Phylogeny of Theaceae by Using Low-Copy and Multi-Copy Nuclear Gene Makers and Uncovers a Fast Radiation Event Contributing to Tea Plants Diversity.2022. 11(7): 1007. **Biology** (**IF=5.168**)

6. Yifan Jiang#,*, Fei Chen#, Aixia Song#, __Yiyong Zhao#__, Xinlu Chen, Yuan Gao, Guo Wei, Wanbo Zhang, Yaqin Guan, Jianyu Fu, Junhao Chen, Wenlei Guo, Zhengjia Wang, Tobias G. Köllner, Zhineng Li, Ning Liu, Shunzhao Sui, Liangsheng Zhang*, Feng Chen*. The genome assembly of Chimonanthus praecox var. concolor and comparative genomic analysis highlight the genetic basis underlying conserved and variable floral traits of wintersweet.2023, 206:117603, **Industrial Crops and Products**. (**IF=5.9**)

7. ZHANG L, ZHU X, **ZHAO Y**, Guo J, ZHANG T, HUANG W, HUANG J, HU Y, HUANG C*, MA H 2022. Phylotranscriptomics Resolves the Phylogeny of Pooideae and Uncovers Factors for Their Adaptive Evolution. **Molecular Biology Evolution** [J], 39(2): msac026. (**IF= 16.24**)

8. GUO J, XU W, HU Y, HUANG J, **ZHAO Y**, ZHANG L, HUANG C-H, MA H 2020. Phylotranscriptomics in Cucurbitaceae reveal multiple whole genome duplications and key morphological and molecular innovations. **Molecular Plant** [J], 13: 1-17. (**IF=13.164**)

9. HE C, CHEN Z, **ZHAO Y**, YU Y, WANG H, WANG C, QI J, WANG Y 2022 Histone demethylase IBM1-mediated meiocyte gene expression ensures meiotic chromosome synapsis and recombination.2022. **Plos Genetics** [J], 18(2): e1010041.(**IF= 5.917**)

10. LIANG Y, WANG S, ZHAO C, MA X, **ZHAO Y**, SHAO J, LI Y, LI H, SONG H, MA H, LI H, ZHANG B, ZHANG L 2020. Transcriptional regulation of bark freezing tolerance in apple (*Malus domestica* Borkh.). **Horticulture Research** [J], 7: 205. (**IF= 6.072**)

11. MENG Z, HAN J, LIN Y, **ZHAO Y**, LIN Q, MA X, WANG J, ZHANG M, ZHANG L, YANG Q, WANG K 2020. Characterization of a *Saccharum spontaneum* with a basic chromosome number of x = 10 provides new insights on genome evolution in genus *Saccharum*. **Theoretical and Applied Genetics** [J], 133: 187-199. (**IF= 5.699**)

12. HUANG W, ZHANG L, COLUMBUS J T, HU Y, **ZHAO Y**, TANG L, GUO Z, CHEN W, MCKAIN M, BARTLETT M, HUANG C-H, LI D-Z, GE S, KELLOGG E A, HONG M 2021. A Well-supported nuclear phylogeny of Poaceae and implications for the evolution of C4 photosynthesis. **Molecular Plant**[J], 15(4): 755-777. (**IF=13.164**)

13. MENG F, LIU L, PENG M, WANG Z, WANG C, **ZHAO Y** 2015. Genetic diversity and population structure analysis in wild strawberry (*Fragaria nubicola* L.) from Motuo in Tibet Plateau based on simple sequence repeats (SSRs). **Biochemical Systematics and Ecology** [J], 63: 113-118. (**IF=1.381**)

14. ZHENG Y, **ZHAO Y**, WU L, LIN J. Copy number variation and diversification of duplicates in *CaCA* gene family of *Capsella*. 2020. **Journal of Fudan University** (Natural Science) [J], 59, No.1:1-31. (In Chinese)

15. HUANG J, XU W, ZHAI J, HU Y, GUO J, ZHANG C, **ZHAO Y**, ZHANG L, MARTINE C, MA H, HUANG C. 2023. Nuclear phylogeny and insights into whole genome duplications and reproductive development of Solanaceae plants. **Plant Communications[J]** DOI:https://doi.org/10.1016/j.xplc.2023.100595  (**IF=8.625**)
