<div align="center">
<img src="Photos/YiyongZhao.jpg" width = "176" height = "256"> 
</div> 

### Curriculum Vitae ([中文版本](./Chinese.md))

<div align="justify">
Yiyong(Rambo) Zhao, born in 1991, currently work as a Postdoctoral Researcher in Bioinformatics at Harvard University. His primary research interests in unveiling the undiscovered functions of the human genome, particularly in the context of neurological diseases. He accomplishes this by applying and refining computational methodologies that incorporate multi-dimensional data from various sources.    

Rambo earned his Ph.D. in Bioinformatics from Fudan University through a joint Ph.D. program supported by the Chinese Scholarship Council at Pennsylvania State University, USA, in the period from 2019 to 2020. Following this, he gained industry experience as a Bioinformatics & AI Research Scientist at Drug Farm, an innovative drug discovery company in Shanghai led by Dr. Tian Xu, a Chair Genetics Professor at Yale and Vice President at Westlake University. From 2021 to 2022, Dr. Zhao primarily engaged in the discovery of indications and biomarkers and participated in the development of a graph neural network-based method for novel drug target identification. This period in the industry led him to understand his deep-rooted passion for academic research, guiding his transition back to academia. In the academic year 2022-2023, Dr. Zhao held a Professorship at Guizhou University, specializing in crop genetic breeding. He was also recognized as a Class A Professor at the College of Agriculture in Guizhou University during the same year.    

Rambo has been engaged in gene mining and bioinformatic studies, with over ten papers published in reputed scientific journals such as Nature and Molecular Plant. The publications were chosen as significant basic research results in Shanghai in 2021 and acknowledged as part of the Chinese Science and Technology Journal Excellence Program, forming an integral groundwork for the conservation of crop germplasm resources and breeding. Furthermore, they also established a solid foundation for genomics in molecular cultivation breeding.Dr. Zhao's contributions extend to academic peer review, having evaluated numerous papers for distinguished journals including Molecular Plant, Horticulture Research, Frontiers in Plant Science, Frontiers Molecular Biosciences, Molecular Phylogenetics and Evolution, and Journal of Systematics and Evolution. From 2022 to the present, he serves as an Editorial Board member of Tropical Plants (ISSN 2833-9851) and as a Peer Reviewer for Innovation (ISSN 2959-5150).    
</div> 

    Email：yiyongzhao16@163.com; yzhao@bwh.harvard.edu
    Mobile：+18573815442 (USA)  
    WechatID：zhaorambo
    Address: 75 Francis ST, Boston, MA, USA.

### Education

2016.06 ~ 2021.06：Ph.D. in Bioinformatics, School of Life Sciences, Fudan University, China 

2019.09 ~ 2020.10：China Scholarship Council (CSC) Joint Ph.D. Program, Department of Biology, Pennsylvania State University, USA

2012.06 ~ 2016.06：Bachelor of Science in Agriculture (*summa cum laude*), Northeast Forestry University, China

2013.06 ~ 2014.01：CSC Joint Bachelor Program, Kangwon National University, South Korea.



### Work Experience

2023.05 ~ now: Postdoctoral Research Fellow, Harvard University, USA.

2022.10 ~ 2023.05: Professor (Special-term Professor Class A), Guizhou University, China

2021.07 ~ 2022.08: Bioinformatics & AI Scientist, at [Drug Farm Co. Ltd](https://drug-farm.com), Shanghai, China.


### Research Interests
<p align = "justify">
Biological big data mining; genomics, phylogenomics and comparative genomics; bioinformatics software development; genome assembly, genome evolution such as whole genome duplications; hybridization detection; gene evolution and functional genomics.
</p>


### Publications (# Co-first author, * Corresponding author, ciatations=473, h-index=8)

1. __ZHAO Y__, ZHANG R, JIANG K, QI J, HU Y, GUO J, ZHU R, ZHANG T, EGANA N, YI T-S, HUANG C-H, MA H 2021. Nuclear phylotranscriptomics/phylogenomics support numerous polyploidization events and hypotheses for the evolution of rhizobial nitrogen-fixing symbiosis in Fabaceae. **Molecular Plant** [J]. 2021, 14(5): 748-773 (First author, **IF= 13.164**, Cover story & Featured article, the study reported in dozens of domestic and international mainstream media such as Guangming Daily, People's Daily, China News, Science Daily, Phy.org, etc., Contribution: for the cover article, based on the transcriptome and genome, we constructed the largest phylogenetic relationship of Leguminosae to date. Phylogenetic relationships were resolved for five subfamilies, about 30 genome-wide replication events were identified, the evolutionary history of the nitrogen-fixing gene family was explored, and the important role of a new gene in nitrogen fixation was proposed, providing an important basis for germplasm conservation and breeding in the legume family.)

2. ZHANG L#\*, CHEN F#, ZHANG X#, LI Z#, **ZHAO Y#**, LOHAUS R#, CHANG X#, DONG W, HO S Y W, LIU X, SONG A, CHEN J, GUO W, WANG Z, ZHUANG Y, WANG H, CHEN X, HU J, LIU Y, QIN Y, WANG K, DONG S, LIU Y, ZHANG S, YU X, WU Q, WANG L, YAN X, JIAO Y, KONG H, ZHOU X, YU C, CHEN Y, LI F, WANG J, CHEN W, CHEN X, JIA Q, ZHANG C, JIANG Y, ZHANG W, LIU G, FU J, CHEN F, MA H, VAN DE PEER Y, TANG H 2020. The water lily genome and the early evolution of flowering plants. **Nature** [J], 577: 79-84. (Co-first author, **IF= 49.962**, ESI highly cited paper, Contribution: Involved in the assembly, annotation and correction of transcriptome data of the first aquatic early angiosperm genome. Based on 115 transcriptome and genome-wide data, a highly supported phylogenetic tree of flowering plants was constructed, and molecular clock analysis revealed that early angiosperms originated in the early Cretaceous, while highly supporting the phylogenetic relationships within the Water Lily family. Low-copy nuclear genes from the whole genome confirmed that saprophytic camphor was the earliest differentiated angiosperm. Meanwhile, using a comparative genomics approach, the Blue Star water lily was identified as a potential hybrid parent of two cultivars, laying the foundation for genomics in molecular cultivation breeding.)

3. Lin Cheng#, Mengge Li#, Yachao Wang#, Qunwei Han, Yanlin Hao, Zhen Qiao, Wei Zhang, Lin Qiu, Andong Gong, Zhihan Zhang, Tao Li, Shanshan Luo, Linshuang Tang, Daliang Liu, Hao Yin, Song Lu, Tiago S.Balbuena and __Yiyong Zhao\*__. Transcriptome-based Variations Effectively Untangling the Intraspecific Relationships and Selection Signals in Xinyang Maojian Tea Population. 2023 Volume 14 doi: 10.3389/fpls.2023.1114284. **Frontiers in Plant Science**(section in Plant Bioinformatics). (**IF=6.627**)

4. CHENG L, CHEN F, HAN Q, LI M, Tiago Santana Balbuena, __ZHAO Y\*__. Phylogenomics as an effective approach to untangle cross-species hybridization event: a case study in the family Nymphaeaceae. 2022. 13. **Frontiers in Genetics**(section in computational genomics). (**IF=4.772**)

5. CHENG L#, LI M#, HAN Q, QIAO Z, HAO Y, Tiago Santana Balbuena, __ZHAO Y\*__. Phylogenomics Resolves the Phylogeny of Theaceae by Using Low-Copy and Multi-Copy Nuclear Gene Makers and Uncovers a Fast Radiation Event Contributing to Tea Plants Diversity.2022. 11(7): 1007. **Biology** (**IF=5.168**)

6. ZHANG L, ZHU X, **ZHAO Y**, Guo J, ZHANG T, HUANG W, HUANG J, HU Y, HUANG C*, MA H 2022. Phylotranscriptomics Resolves the Phylogeny of Pooideae and Uncovers Factors for Their Adaptive Evolution. **Molecular Biology Evolution** [J], 39(2): msac026. (**IF= 16.24**)

7. GUO J, XU W, HU Y, HUANG J, **ZHAO Y**, ZHANG L, HUANG C-H, MA H 2020. Phylotranscriptomics in Cucurbitaceae reveal multiple whole genome duplications and key morphological and molecular innovations. **Molecular Plant** [J], 13: 1-17. (**IF=13.164**)

8. HE C, CHEN Z, **ZHAO Y**, YU Y, WANG H, WANG C, QI J, WANG Y 2022 Histone demethylase IBM1-mediated meiocyte gene expression ensures meiotic chromosome synapsis and recombination.2022. **Plos Genetics** [J], 18(2): e1010041.(**IF= 5.917**)

9. LIANG Y, WANG S, ZHAO C, MA X, **ZHAO Y**, SHAO J, LI Y, LI H, SONG H, MA H, LI H, ZHANG B, ZHANG L 2020. Transcriptional regulation of bark freezing tolerance in apple (*Malus domestica* Borkh.). **Horticulture Research** [J], 7: 205. (**IF= 6.072**)

10. MENG Z, HAN J, LIN Y, **ZHAO Y**, LIN Q, MA X, WANG J, ZHANG M, ZHANG L, YANG Q, WANG K 2020. Characterization of a *Saccharum spontaneum* with a basic chromosome number of x = 10 provides new insights on genome evolution in genus *Saccharum*. **Theoretical and Applied Genetics** [J], 133: 187-199. (**IF= 5.699**)

11. HUANG W, ZHANG L, COLUMBUS J T, HU Y, **ZHAO Y**, TANG L, GUO Z, CHEN W, MCKAIN M, BARTLETT M, HUANG C-H, LI D-Z, GE S, KELLOGG E A, HONG M 2021. A Well-supported nuclear phylogeny of Poaceae and implications for the evolution of C4 photosynthesis. **Molecular Plant**[J], 15(4): 755-777. (**IF=13.164**)

12. MENG F, LIU L, PENG M, WANG Z, WANG C, **ZHAO Y** 2015. Genetic diversity and population structure analysis in wild strawberry (*Fragaria nubicola* L.) from Motuo in Tibet Plateau based on simple sequence repeats (SSRs). **Biochemical Systematics and Ecology** [J], 63: 113-118. (**IF=1.381**)

13. ZHENG Y, **ZHAO Y**, WU L, LIN J. Copy number variation and diversification of duplicates in *CaCA* gene family of *Capsella*. 2020. **Journal of Fudan University** (Natural Science) [J], 59, No.1:1-31. (In Chinese)

14. HUANG J, XU W, ZHAI J, HU Y, GUO J, ZHANG C, **ZHAO Y**, ZHANG L, MARTINE C, MA H, HUANG C. 2023. Nuclear phylogeny and insights into whole genome duplications and reproductive development of Solanaceae plants. **Plant Communications[J]** DOI:https://doi.org/10.1016/j.xplc.2023.100595  (**IF=8.625**)  
